tests/smoke-tests/src/test/java/org/apache/activemq/artemis/tests/smoke/logging/AuditLoggerTest.java

```diff
@@ -117,6 +121,56 @@ public void testAuditLog() throws Exception {
       checkAuditLogRecord(true, "is sending a message");
    }
 
+   @Test
+   public void testCoreConnectionAuditLog() throws Exception {
+      testConnectionAuditLog("CORE");
+   }
+
+   @Test
+   public void testAMQPConnectionAuditLog() throws Exception {
+      testConnectionAuditLog("AMQP");
+   }
+
+   @Test
+   public void testOpenWireConnectionAuditLog() throws Exception {
+      testConnectionAuditLog("OPENWIRE");
+   }
```

Comment:
Since these are considered 'resource' logs, they might be better in AuditLoggerResourceTest